src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    {
+        radius = 10;
+        setColor("White");
+    }
```

Comment:
You don't need this block, just make a constructor in which you use the keyword super() for color. Also add constructors to all figure classes and then use them to create figure

Reply:
Added constructors for all clases